<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1417</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(44, 170, 166, 255), stop:1 rgba(0, 0, 91, 255));
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="tabSelections">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="summaryBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>1061</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 179, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(44, 170, 166, 0), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>The Editor's Dream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearSummary">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(44, 170, 166, 255), stop:1 rgba(248, 248, 248, 0));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CLEAR TEXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(44, 170, 166, 0), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SUMMARY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(44, 170, 166, 255), stop:1 rgba(248, 248, 248, 0));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quickSummary">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(44, 170, 166, 255), stop:1 rgba(248, 248, 248, 0));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>QUICK SUMMARY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1417</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionGenerateReport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionGenerateReport">
   <property name="text">
    <string>Generate Report</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
